heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -232,6 +237,20 @@ function process_message()
             if type(info.subsessionCounter) == "number" then
                 msg.Fields.subsessionCounter.value = info.subsessionCounter
             end
+
+            if type(info.subsessionStartDate) == "string" then
+                local t = dt.rfc3339:match(info.subsessionStartDate)
```

Comment:
This timestamp is a little screwed up (some values aren't ISO as spec'ed). I am thinking we may just want the date part (treat it UTC and ignore the timezone). i.e. just keep the else clause.

`2015-08-05T00:00:00.0-07:00`
`2015-08-05T00:00:00`